### PR TITLE
Add vote icon (#898)

### PR DESCRIPTION
**Pull request by ptrgast**
Hello! Here is a proposal for the vote icon.

preview-zoomed

### CODE REVIEW

**Comment by karsa-mistmere** [comment]
Hey @ptrgast,

Nice work, but I'm afraid this design has multiple guideline violations, please refer to our [official documentation](https://lucide.dev/docs/icon-design-guide).

**Comment by ptrgast** [comment]
Thanks for the response, I made a revision following the guidelines more carefully. What do you think about it?

revision1-preview-zoomed

File: icons/vote.svg [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+  viewBox="0 0 24 24"
+  fill="none"
+  stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+	<path d="m6 14v8h-2 18-2v-12h-3"/>
+	<path d="m2 2h11c7 0 7 4 7 4h-5c-3 0-5 2-5 2l2 2c2 2 0 4-2 2l-6-6h-2"/>
+	<path d="m10 17 2 2 4-5"/>
```
Comment (karsa-mistmere) [inline icons/vote.svg:14]: Nice, that's a much cleaner design, I think we can go with this one, but we should definitely use the existing checkmark as it appears on `file-check`, `verified` etc.
image

```suggestion
  <path d="m9 12 2 2 4-4" />
  <path d="M5 7c0-1.1.9-2 2-2h10a2 2 0 0 1 2 2v12H5V7Z" />
  <path d="M22 19H2" />
```

Comment (ptrgast) [reply icons/vote.svg:14]: Very good point, I will put them all together.

Comment (karsa-mistmere) [reply icons/vote.svg:14]: I’ve already done that for you, hence the suggestion. :)

Comment (ptrgast) [reply icons/vote.svg]: That's great! (I thought you only added the checkmark earlier...)

**Review by ericfennis — APPROVED** [review]

**Pull request merged**